### ISSUE TITLE
[BUG] ./gradlew build -PadminChassisOnly does not build server-chassis-spring causing error

### ISSUE THREAD

**Issue by ilovechai** [odpi/egeria#7492]
### Current Behavior

On executing from root of the project:
` ./gradlew build -PadminChassisOnly` 
gradle gives the following error:

```
egeria/open-metadata-implementation/adapters/open-connectors/connector-configuration-factory/src/main/java/org/odpi/openmetadata/adapters/repositoryservices/ConnectorConfigurationFactory.java:6: error: package org.odpi.openmetadata.adapters.eventbus.topic.kafka does not exist
import org.odpi.openmetadata.adapters.eventbus.topic.kafka.KafkaOpenMetadataTopicProvider;
```

On removing the unused import:
```
import org.odpi.openmetadata.adapters.eventbus.topic.kafka.KafkaOpenMetadataTopicProvider;
```
the server-chassis-spring build is sucessful

### Expected Behavior

server-chassis-spring must be able to build sucessfully

### Steps To Reproduce

```
` ./gradlew build -PadminChassisOnly` 
```

### Environment

```markdown
- Egeria:3.15
- OS:Mac
- Java:15
```

**Comment by planetf1** [comment]
The build failure occured when using the `-PadminChassisOnly` option, and only if a regular build had not been done first, and `clean` was not used. Additionally this option is not tested as part of the formal build process.

**Comment by ilovechai** [comment]
Closing as PR is merged into `main` and `egeria-release-3.15` and issue is resolved.
